pkg/apis/build/v1alpha1/buildstrategy_types.go

```diff
@@ -84,6 +84,11 @@ func (s BuildStrategy) GetVolumes() []BuildStrategyVolume {
 	return s.Spec.Volumes
 }
 
+// GetLabels returns the labels the build strategy
+func (s BuildStrategy) GetLabels() map[string]string {
+	return s.Labels
+}
```

Comment:
I don't think this is needed because Kubernetes already provides that function on `ObjectMeta`.